Podfile

```diff
@@ -0,0 +1,32 @@
+platform :ios, '12.0'
+use_frameworks!
+
+target 'Example' do
+	pod 'NYTPhotoViewer', :path => '.'
+end
+
+target 'Example-Swift' do
+end
+
+target 'NYTPhotoViewer' do
+  pod 'PINRemoteImage/iOS', '~> 3.0.1'
```

Comment:
Seems weird to use the optimistic version specifier `~>` to a specific 3-digit version number. I'd prefer `~> 3.0` or even just leave off the version entirely, since that's what the lockfile is for.

Reply:
good call, i removed it.